Frends.GooglePubSub.Publish/Frends.GooglePubSub.Publish.Tests/Frends.GooglePubSub.Publish.Tests.cs

```diff
@@ -0,0 +1,52 @@
+using NUnit.Framework;
+using System;
+using System.Linq;
+using System.Threading.Tasks;
+using Frends.GooglePubSub.Publish.Definitions;
+
+namespace Frends.GooglePubSub.Tests
+{
+    [TestFixture]
+    class Tests
+    {
+        private const string TestProjectId = "my-project";
+        private const string TestTopicId = "my-topic";
+
+        [SetUp]
+        public void SetUp()
+        {
+            Environment.SetEnvironmentVariable("PUBSUB_EMULATOR_HOST", "localhost:8681");
+        }
+
+        [Test]
+        public async Task PublishTest()
+        {
+            var result = await GooglePubSub.Publish.GooglePubSub.Publish(new Input
```

Comment:
Method call can be simplified to `Publish.GooglePubSub.Publish`, so remove `GooglePubSub` from the beginning.